kofta/src/vscode-webview/atoms.ts

```diff
@@ -3,88 +3,101 @@ import { useCurrentRoomStore } from "../webrtc/stores/useCurrentRoomStore";
 import { Room, BaseUser, UserWithFollowInfo } from "./types";
 
 const createSetter = <T>(a: WritableAtom<T, any>) =>
-  atom(null, (get, set, fn: (x: T) => T) => {
-    set(a, typeof fn === "function" ? fn(get(a)) : fn);
-  });
+	atom(null, (get, set, fn: (x: T) => T) => {
```

Comment:
use og spacing

(is prettier hook doing this?)

Reply:
I think I reinstalled the project, and have to run `npm install` in root so husky register the git hook.

Reply:
shouldn't the hook only format files you touch? Also can we configure it to use 2 spaces

Reply:
@benawad it does it project wide

Reply:
we should move husky to `kofta`, the point of the hook is to style other contributors commits that don't know the format yet, but I doubt many of them will `npm i` at the root